### PR TITLE
Vt creating blog repo

### LINKED ISSUE #36: Add Blog
As a blog connoisseur I would like to add a blog to my list of favorites, so that I can keep track of new blogs I encounter.

An Blog should have:

- A Title
- A URL

**Given** the user is viewing the Blog Management menu  
**When** they select the option to add a blog  
**Then** they should be prompted to enter the new blog's title and url  
**Then** the new blog should be saved to the database

### PR DESCRIPTION
**Pull request by vishus12**
# Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

-Created Blog Manager
-Created Blog Repository
-Added the Insert Method to repo
-Created the Add Method to Manager

Fixes # (issue)
Closes #36

## Type of change
Please delete options that are not relevant.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
# Testing Instructions
Please describe the tests required to verify your changes. Provide instructions so PR Tester can check functionality. Please also list any relevant details for your tests
# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings or errors
- [ ] I have added test instructions that prove my fix is effective or that my feature works

### CODE REVIEW

**Review by jandershock — APPROVED** [review]
I think the .vscode folder needs to not be added to main?

**Comment by vishus12** [comment]
added another commit. My Blogmanager was deleted off of main menu manager when I pulled jordans code. I just fixed it

**Review by jandershock — APPROVED** [review]
:shipit:

File: TabloidCLI/Repositories/BlogRepository.cs [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        public List<Blog> GetAll()
+       {
+            using (SqlConnection conn = Connection)
+            {
+                conn.Open();
+              using (SqlCommand cmd = conn.CreateCommand())
+               {
+                  cmd.CommandText = @"";
+                }
+                return GetAll();
```
Comment (joshdbarton) [inline TabloidCLI/Repositories/BlogRepository.cs:34]: pretty sure this will cause a stack overflow if you called it...

**Review by joshdbarton — CHANGES_REQUESTED** [review]

File: TabloidCLI/Repositories/BlogRepository.cs (hunk 1 above)
Comment (joshdbarton) [reply TabloidCLI/Repositories/BlogRepository.cs:34]: replace the code in the method with `throw new NotImplementedException();` like what you have in the rest of the methods.

**Review by ToryT23 — APPROVED** [review]
looks good!

**Review by joshdbarton — APPROVED** [review]
:shipit:

File: TabloidCLI/UserInterfaceManagers/MainMenuManager.cs [hunk 2]
```diff
@@ -26,7 +26,7 @@ public IUserInterfaceManager Execute()
                 case "1": return new JournalManager(this, CONNECTION_STRING);
                 case "2": throw new NotImplementedException();
                 case "3": return new AuthorManager(this, CONNECTION_STRING);
-                case "4": throw new NotImplementedException();
+                case "4": return new BlogManager(this, CONNECTION_STRING);
```
Comment (jandershock) [inline TabloidCLI/UserInterfaceManagers/MainMenuManager.cs:29]: This should be case 2 😝

Comment (vishus12) [reply TabloidCLI/UserInterfaceManagers/MainMenuManager.cs:29]: thats an ez fix jordan

**Pull request merged**